pkg/minikube/config/types.go

```diff
@@ -35,6 +35,7 @@ type MachineConfig struct {
 	CPUs                int
 	DiskSize            int
 	VMDriver            string
+	ContainerRuntime    string
```

Comment:
1. this duplicates the config between `MachineConfig` and `KubernetesConfig`  
2. if there is no way to get rid of this duplication, and this is really the best way to propagate this flag to the driver, I'd comment similar to the other fields of this struct `// only used by the None driver`

Reply:
KubernetesConfig is too late, since the provisioning is done before the bootstrapping